### ISSUE TITLE
Feature request: Happiness tracker

### ISSUE THREAD

**Issue by PabloBenigno** [microsoft/vsts-extension-retrospectives#7]
When you have a retrospective, you also ask all the team for a happiness value (whatever scale you use 1-5 1-10...).

As a Scrum Master I want to keep track of the happiness of the team members.
I want to be able to set a happiness value on every retrospective.
I want to be able to see the history of the happiness of a team member.
I want to be able to see the history of several team members at the same time.
I want to be able to see the history of a team member happiness and compare it with the team average.

Step by step, it would be nice to have this feature.

**Comment by polatengin** [comment]
I liked the idea.

May I ask kindly to elaborate it a bit, such as, how you measure, when you measure, do you have comparison based on the history, is it anonymous or not, etc.

**Comment by PabloBenigno** [comment]
What we usually do is, every retrospective, every team member gives a happiness value from 1 to 10.
1 means -> This is all crap and I'm leaving the company
...
5 means -> I'm ok, but there are things I'm unhappy with
...
10 means -> I'm super happy
-------------
The scale is not really important, as each team can handle it in different ways. Numeric, stars, whatever works for a team.

So, every team member gives this value, that MUST be honest, because it makes no sense on giving a top value if you're not feeling that good.

The point is that if you see that someone is getting a lower happiness every sprint, that's an alarm for a Scrum Master, so you can contact that team member and try to figure out what's wrong and try to act to solve it.

Having a history of the happiness of every team member would help to see that.

**Comment by polatengin** [comment]
thanks for the clarification, if I understand it correctly, lowest value means

> this project or this sprint is not the best, I'm not happy with it

right? it shouldn't be about the company, cause retro is not about the company, it should be about the sprint or the project.

my understanding is, if something came up to surface during retro, there should be an actionable item. then, this voting should be done before closing the retro, so scrum master can take a look at it and create some action items if needed.

**Comment by polatengin** [comment]
Though it's a very good feature request, I think this feature shouldn't be part of the Retrospective extension. It should be part of another extension, I believe.

I'm closing this issue, but please continue to add comments if you still think this feature should be part Retro extension.

**Comment by HamzaBoukraa** [comment]
Why not use a similar concept as a Niko-Niko?

It will track the mood of the team all over the sprint and not only in the retrospective day.

**Comment by JankaKoen** [comment]
Thank you very much for your feedback. The feature is currently not something we are planning on implementing, but we will continue to track these comments in case this changes
